### PR TITLE
Improve nushell template to gracefully handle no match errors

### PR DESCRIPTION
**Pull request by marcelarie**
Currently, the `__zoxide_z` command in nushell throws the following error when no matches are found for a query:
```shell
zoxide: no match found
Error: nu::shell::non_zero_exit_code

  × External command had a non-zero exit code
    ╭─[/home/marcel/.config/nushell/external/zoxide.nu:33:8]
 32 │   
 33 │       (zoxide query --exclude $env.PWD -- ...$rest | str trim -r -c "\n")
    ·        ───┬──
    ·           ╰── exited with code 1
 34 │     
    ╰────
```

when the behavior in any other shell is to only return the result of the zoxide command:
```shell
zoxide: no match found
```

This PR modifies the nushell template to ignore errors and display only the `zoxide query` command result. The cd command is executed only when a valid path is found.

### CODE REVIEW

File: templates/nushell.txt [hunk 1]
```diff
@@ -45,9 +45,16 @@ def --env --wrapped __zoxide_z [...rest:string] {
...
-    (zoxide query --exclude $env.PWD -- ...$rest | str trim -r -c "\n")
+    (try {
+      zoxide query --exclude $env.PWD -- ...$rest | str trim -r -c "\n"
+    } catch {
+        "" 
+    })
+  }
+
+  if $path != "" {
+    cd $path
```
Comment (marcelarie) [inline templates/nushell.txt:56]: I am happy to change the formatting if any other is preferred.

**Comment by ajeetdsouza** [comment]
Thank you!!

**Comment by ajeetdsouza** [comment]
image

I'm already seeing the expected behaviour on Nu 0.98.0 (macOS). On what version are you seeing the error trace?

**Comment by marcelarie** [comment]
> I'm already seeing the expected behaviour on Nu 0.98.0 (macOS). On what version are you seeing the error trace?

`0.99` after an update, everything is working now. Maybe it was something related to my config 🥲

**Pull request closed without merging**